package.json

```diff
@@ -4,9 +4,12 @@
   "description": "A client embedding Daily Prebuilt, optimized for TestRTC tests",
   "scripts": {
     "build": "webpack --mode=production",
-    "start": "http-server ./dist/ -c-1",
+    "dev": "sh ./dev.sh",
+    "netlify-build": "npm run netlify-config-redirect && npm run build",
+    "netlify-config-redirect": "sed -i.bak s/DAILY_API_KEY_PLACEHOLDER/${DAILY_API_KEY}/g netlify.toml",
```

Comment:
I love this line, perfect combination of simple and bash-y. Could this break on a windows user's laptop though or is this only called when something is being deployed?

Reply:
Yeah, the dev flow would currently not work on Windows (`npm run dev` also involves running a bash script!)

I will add a note about this to the README! If we have Windows users who are not using WSL we can add a flow for Windows as well. Right now I have no reliable way to test something like that since my Window machine is with the repair company after drinking a glass of water >.>